cli/hzn.go

```diff
@@ -185,10 +185,10 @@ Environment Variables:
 	exNodeAddPolicyIdTok := exNodeAddPolicyCmd.Flag("node-id-tok", msgPrinter.Sprintf("The Horizon Exchange node ID and token to be used as credentials to query and modify the node resources if -u flag is not specified. HZN_EXCHANGE_NODE_AUTH will be used as a default for -n. If you don't prepend it with the node's org, it will automatically be prepended with the -o value.")).Short('n').PlaceHolder("ID:TOK").String()
 	exNodeAddPolicyNode := exNodeAddPolicyCmd.Arg("node", msgPrinter.Sprintf("Add or replace policy for this node.")).Required().String()
 	exNodeAddPolicyJsonFile := exNodeAddPolicyCmd.Flag("json-file", msgPrinter.Sprintf("The path of a JSON file containing the metadata necessary to create/update the node policy in the Horizon exchange. Specify -f- to read from stdin.")).Short('f').Required().String()
-	exNodeUpdatePolicyCmd := exNodeCmd.Command("updatepolicy", msgPrinter.Sprintf("Update an attribute of the policy for this node in the Horizon Exchange."))
+	exNodeUpdatePolicyCmd := exNodeCmd.Command("updatepolicy", msgPrinter.Sprintf("(Deprecated) Update an attribute of the policy for this node in the Horizon Exchange. The input should contain either the node properties or the constaints, not both."))
```

Comment:
Please update the message:
"(DEPRECATED) This command is deprecated. Please use 'hzn exchange node addpolicy' to update the node policy. This command is used to update either the node policy properties or the constraints, but not both."

Reply:
done